documentation/source/users/rmg/installation/index.rst

```diff
@@ -21,12 +21,19 @@ Installation on a Windows Platform
 
 Due to difficulties with dependencies, installation on Windows directly is no longer supported. Instead, it is
 recommended to run a Linux virtual machine from Windows and follow either the instructions for basic users
-(binary installation using Anaconda) or the instructions for developers.
+(binary installation using Anaconda) or the instructions for developers. Alternatively, it is also possible to install
+RMG in the Ubuntu subsystem now available on Windows 10.
 
 .. toctree::
     :maxdepth: 1
 
     virtualMachineSetup
+    linuxSubsystem
+
+For users unfamiliar with bash or Linux, we recommend looking at online tutorials such as the one
+`here <https://www.guru99.com/unix-linux-tutorial.htmlhttps://www.guru99.com/unix-linux-tutorial.html>`_. To start out
```

Comment:
Is the link here duplicated on purpose?
(could you compile the documentation and make sure all links work?)

Reply:
No, this is a mistake. Thanks for the catch!

Reply:
I checked the remaining links and they all seemed to work. I am pretty sure that I checked them before, but most browsers are also able to deal with this duplication properly. I also looked through the code, and this was the only duplication I found.